src/main/webapp/app/shared/utils/LodashUtils.ts

```diff
@@ -0,0 +1,273 @@
+type DebouncedFunction<F extends (...args: any[]) => any> = (
```

Comment:
This file needs to be reviewed thoroughly. I tried to create some tests to cover each method in Lodashutils.spec.ts. But I might have missed some edge cases.

Reply:
I also hope to move this file to cbioportal-frontend repo so we can retire the lodash dependency from cbioportal-frontend-commons, react-mutation-mapper repos.

Reply:
Some methods are from https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore, I filled in the types. The rest are created by me.

Reply:
Interestingly I would expect it to fail within the usage in FormValidationUtils. The args are spread to the `debouncedCompanyNameValidator` which have different types. Did I misunderstand the use of `unknown`?

Reply:
`unknown` means you have no idea what the type is (so it could be a union type which is your case). If anything cares about the type then you have to prove it's the expected type with type checks.
`any` means turn off type checking.

Reply:
Doesn't unknown forbid unknown to be assigned to other type? https://stackoverflow.com/a/51439876/1868022

Reply:
Let's talk about it on a call. I don't feel like I can explain it well in a comment.